src/System.Runtime/tests/System/TypeTests.netcoreapp.cs

```diff
@@ -199,5 +200,49 @@ public static IEnumerable<object[]> NotDefinedTypes
                 yield return new object[] { theT };
             }
         }
+
+        [Theory]
+        [MemberData(nameof(IsByRefLikeTestData))]
+        public static void TestIsByRefLike(Type type, bool expected)
+        {
+            Assert.Equal(expected, type.IsByRefLike);
+        }
+
+        public static IEnumerable<object[]> IsByRefLikeTestData
+        {
+            get
+            {
+                Type theT = typeof(Outside<>).GetTypeInfo().GenericTypeParameters[0];
+
+                yield return new object[] { typeof(TypedReference), true };
+                yield return new object[] { typeof(Span<>), true };
+                yield return new object[] { typeof(Span<int>), true };
+                yield return new object[] { typeof(Span<>).MakeGenericType(theT), true };
+                yield return new object[] { typeof(ByRefLikeStruct), true };
+                yield return new object[] { typeof(RegularStruct), false };
+                yield return new object[] { typeof(object), false };
+                yield return new object[] { typeof(Span<int>).MakeByRefType(), false };
+                yield return new object[] { typeof(Span<int>).MakePointerType(), false };
+                yield return new object[] { theT, false };
+                yield return new object[] { typeof(int[]), false };
+                yield return new object[] { typeof(int[,]), false };
+                yield return new object[] { Type.GetTypeFromCLSID(default(Guid)), false };
+            }
```

Comment:
Is `typeof(int).MakeByRefType()` by-ref like? Ideally it would be, IMO, but either way I think it would be good to be included in the test.

Reply:
So it does. Maybe that's too subtle, or maybe it's just time for me to go to bed 😆